tools/middlewares.go

```diff
@@ -5,8 +5,8 @@ import (
 	"TUM-Live/model"
 	"errors"
 	"github.com/getsentry/sentry-go"
-	"github.com/gin-contrib/sessions"
 	"github.com/gin-gonic/gin"
+	"github.com/golang-jwt/jwt"
```

Comment:
```suggestion
	"github.com/golang-jwt/jwt/v4"
```
As per [github.com/golang-jwt/jwt/v4](https://github.com/golang-jwt/jwt/blob/main/MIGRATION_GUIDE.md). I saw that this deprecates `*jwt.StandardClaims` for example. This should get taken a look at, in case we want to actually update `jwt`.